common-compose.yml

```diff
@@ -0,0 +1,36 @@
+# MySQL designed for container-native deployment on Joyent's Triton platform.
+
+mysql:
+    image: 0x74696d/mysql:latest
+    mem_limit: 4g
+    restart: always
+    env_file: .env
```

Comment:
> you're less likely to copy them around unintentionally

I know, and they're less visible to people when casually browsing, but then somebody tars up the directory and sends it to Shifty Sally/Steve because they didn't see it and realize credentials were in there....